Cargo.lock

```diff
@@ -268,7 +297,7 @@ dependencies = [
  "alloy-json-rpc",
  "alloy-network",
  "alloy-network-primitives",
- "alloy-primitives",
+ "alloy-primitives 0.7.7",
```

Comment:
This is a duplicate dep alongside with alloy-primitives 0.8.10

Reply:
i don't know how that happened tbh

Reply:
came with merging nightly